ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/json/ExecutableDataDeserializer.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.ethereum.core.json;
+
+import org.hyperledger.besu.datatypes.Address;
+import org.hyperledger.besu.datatypes.Hash;
+import org.hyperledger.besu.ethereum.core.ExecutableData;
+import org.hyperledger.besu.ethereum.core.Transaction;
+
+import java.math.BigInteger;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.ObjectCodec;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.deser.std.StdDeserializer;
+import org.apache.tuweni.bytes.Bytes;
+import org.apache.tuweni.units.bigints.UInt64;
+
+public class ExecutableDataDeserializer extends StdDeserializer<ExecutableData> {
+
+  public ExecutableDataDeserializer() {
+    this(null);
+  }
+
+  public ExecutableDataDeserializer(final Class<?> vc) {
+    super(vc);
+  }
+
+  @Override
+  public ExecutableData deserialize(
+      final JsonParser parser, final DeserializationContext deserializer)
+      throws java.io.IOException {
+    ExecutableData executableData = new ExecutableData();
+    ObjectCodec codec = parser.getCodec();
+    JsonNode node = codec.readTree(parser);
```

Comment:
Just me - I have other `node` associations 
```suggestion
    JsonNode json = codec.readTree(parser);
```

Reply:
I think node makes more sense here. In jackson library there are many objects starting with Json i.e JsonTree JsonParser, JsonObject etc.